datalab/datalab_session/util.py

```diff
@@ -27,11 +27,14 @@ def add_file_to_bucket(item_key: str, path: object) -> str:
   log.info(f'Adding {item_key} to {settings.DATALAB_OPERATION_BUCKET}')
 
   s3 = boto3.client('s3')
-  response = s3.upload_file(
-    path,
-    settings.DATALAB_OPERATION_BUCKET,
-    item_key
-  )
+  try:
+    response = s3.upload_file(
+      path,
+      settings.DATALAB_OPERATION_BUCKET,
+      item_key
+    )
+  except ClientError as e:
+    raise ClientError(f'Error uploading the operation output')
```

Comment:
perhaps log the actual exception in case we can get more info there when debugging an issue in the future.